### ISSUE TITLE
Flutter app does not build against iOS simulator

### ISSUE THREAD

**Issue by SeanOThomas** [flutter/flutter#53964]
## Steps to Reproduce

1. ...`flutter clean`
2. ...`open ios/Runner.xcworkspace/`
3. ...cmd-b to build debug against simulator

The steps above are just one example. I've tried various combinations with the following commands as well:

```
rm -rf ios/Pods ios/Podfile.lock
flutter clean
rm -rf  ~/Library/Developer/Xcode/DerivedData
Flutter clean
```

## Logs

Note that I haven't been able to run `flutter build ios` because that uses release, not debug.
I've seen the below error for different Pods, not just `SDWebImage`.  

```
/.../ios/Pods/SDWebImage/SDWebImage/Core/SDWebImagePrefetcher.h:92:1: Property with 'retain (or strong)' attribute must be of object type
```

```
[✓] Flutter (Channel master, v1.16.4-pre.88, on Mac OS X 10.15.3 19D76, locale en-NZ)
    • Flutter version 1.16.4-pre.88 at /Users/sthomas/flutter
    • Framework revision ab14307e0c (17 hours ago), 2020-04-03 00:56:01 -0400
    • Engine revision f5127cc07a
    • Dart version 2.8.0 (build 2.8.0-dev.19.0 fae35fca47)


[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
    • Android SDK at /Users/sthomas/Library/Android/sdk
    • Platform android-29, build-tools 28.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.4)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.4, Build version 11E146
    • CocoaPods version 1.9.1

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 3.5)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 43.0.1
    • Dart plugin version 191.8593
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)

[✓] Connected device (3 available)
    • AOSP on IA Emulator • emulator-5554 • android-x86    • Android 9 (API 28) (emulator)
    • Chrome              • chrome        • web-javascript • Google Chrome 80.0.3987.149
    • Web Server          • web-server    • web-javascript • Flutter Tools

• No issues found!
```

**Comment by iapicca** [comment]
Hi @SeanOThomas 
could you please run `flutter upgrade --force`
and provide your upgraded `flutter doctor -v`;

I'm didn't grasp with which command are you experiencing the issue
according with your case could you provide your `flutter run --verbose` 
or `flutter build ios --verbose`  or `flutter run  --release --verbose`

Thank you

**Comment by SeanOThomas** [comment]
Thank you for the response.  

**Flutter doctor -v**

```
➜  nothing_new git:(master) ✗ flutter doctor -v
[✓] Flutter (Channel master, v1.18.1-pre.15, on Mac OS X 10.15.3 19D76, locale en-NZ)
    • Flutter version 1.18.1-pre.15 at /Users/sthomas/flutter
    • Framework revision 17079f26b5 (6 hours ago), 2020-04-06 01:21:01 -0400
    • Engine revision 9b8dcc7ecf
    • Dart version 2.8.0 (build 2.8.0-dev.20.0 05103dfe5a)

 
[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
    • Android SDK at /Users/sthomas/Library/Android/sdk
    • Platform android-29, build-tools 28.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.4)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.4, Build version 11E146
    • CocoaPods version 1.9.1

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 3.5)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 43.0.1
    • Dart plugin version 191.8593
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)

[✓] Connected device (3 available)
    • AOSP on IA Emulator • emulator-5554 • android-x86    • Android 9 (API 28) (emulator)
    • Chrome              • chrome        • web-javascript • Google Chrome 80.0.3987.149
    • Web Server          • web-server    • web-javascript • Flutter Tools
```

Not sure how much you want to see of flutter build ios --verbose. It's very long.  But here's hte latter half:

```
CONFIGURATION Release
                        ===
                    
                        Check dependencies
                    
                        Write auxiliary files
                        /bin/mkdir -p

/Users/sthomas/Library/Developer/Xcode/DerivedData/Runner-ftvesijaqdysdeapgq

mlgudlnwhc/Build/Intermediates.noindex/Pods.build/Release-iphoneos/leveldb-l
                        ibrary.build/DerivedSources
                        write-file

/Users/sthomas/Library/Developer/Xcode/DerivedData/Runner-ftvesijaqdysdeapgq

mlgudlnwhc/Build/Intermediates.noindex/Pods.build/Release-iphoneos/leveldb-l
                        ibrary.build/DerivedSources/leveldb_vers.c
                        write-file

/Users/sthomas/Library/Developer/Xcode/DerivedData/Runner-ftvesijaqdysdeapgq

mlgudlnwhc/Build/Intermediates.noindex/Pods.build/Release-iphoneos/leveldb-l
                        ibrary.build/module.modulemap
                        /bin/mkdir -p

/Users/sthomas/Library/Developer/Xcode/DerivedData/Runner-ftvesijaqdysdeapgq

mlgudlnwhc/Build/Intermediates.noindex/Pods.build/Release-iphoneos/leveldb-l
                        ibrary.build/Objects-normal/arm64
                        write-file

/Users/sthomas/Library/Developer/Xcode/DerivedData/Runner-ftvesijaqdysdeapgq

mlgudlnwhc/Build/Intermediates.noindex/Pods.build/Release-iphoneos/leveldb-l
                        ibrary.build/Objects-normal/arm64/leveldb.LinkFileList
                    
                        Create product structure
                        /bin/mkdir -p

/Users/sthomas/Desktop/Flutter/nothing_new/build/ios/Release-iphoneos/leveld
                        b-library/leveldb.framework
                        /bin/mkdir -p

/Users/sthomas/Desktop/Flutter/nothing_new/build/ios/Release-iphoneos/leveld
                        b-library/leveldb.framework/Headers
                    
                        ProcessInfoPlistFile

/Users/sthomas/Desktop/Flutter/nothing_new/build/ios/Release-iphoneos/leveld
                        b-library/leveldb.framework/Info.plist Target\ Support\
                        Files/leveldb-library/leveldb-library-Info.plist
                            cd
/Users/sthomas/Desktop/Flutter/nothing_new/ios/Pods
                            export

PATH="/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.plat

form/Developer/usr/bin:/Applications/Xcode.app/Contents/Developer/usr/bi

n:/Users/sthomas/Desktop/firebase/y/google-cloud-sdk/bin:/usr/local/bin:

/usr/bin:/bin:/usr/sbin:/sbin:/Library/Apple/usr/bin:/Users/sthomas/flut
                            ter/bin"
                            builtin-infoPlistUtility

/Users/sthomas/Desktop/Flutter/nothing_new/ios/Pods/Target\ Support\
                            Files/leveldb-library/leveldb-library-Info.plist
-expandbuildsettings
                            -format binary -platform iphoneos
-requiredArchitecture arm64 -o

/Users/sthomas/Desktop/Flutter/nothing_new/build/ios/Release-iphoneos/le
                            veldb-library/leveldb.framework/Info.plist
                    
                        CpHeader leveldb-library/include/leveldb/c.h

/Users/sthomas/Desktop/Flutter/nothing_new/build/ios/Release-iphoneos/leveld
                        b-library/leveldb.framework/Headers/c.h
                            cd
/Users/sthomas/Desktop/Flutter/nothing_new/ios/Pods
                            export

PATH="/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.plat

form/Developer/usr/bin:/Applications/Xcode.app/Contents/Developer/usr/bi

n:/Users/sthomas/Desktop/firebase/y/google-cloud-sdk/bin:/usr/local/bin:

/usr/bin:/bin:/usr/sbin:/sbin:/Library/Apple/usr/bin:/Users/sthomas/flut
                            ter/bin"
                            builtin-copy -exclude .DS_Store -exclude CVS
-exclude .svn -exclude .git
                            -exclude .hg -resolve-src-symlinks

/Users/sthomas/Desktop/Flutter/nothing_new/ios/Pods/leveldb-library/incl
                            ude/leveldb/c.h

/Users/sthomas/Desktop/Flutter/nothing_new/build/ios/Release-iphoneos/le
                            veldb-library/leveldb.framework/Headers
                    
                        CpHeader leveldb-library/include/leveldb/cache.h

/Users/sthomas/Desktop/Flutter/nothing_new/build/ios/Release-iphoneos/leveld
                        b-library/leveldb.framework/Headers/cache.h
                            cd
/Users/sthomas/Desktop/Flutter/nothing_new/ios/Pods
                            export

PATH="/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.plat

form/Developer/usr/bin:/Applications/Xcode.app/Contents/Developer/usr/bi

n:/Users/sthomas/Desktop/firebase/y/google-cloud-sdk/bin:/usr/local/bin:

/usr/bin:/bin:/usr/sbin:/sbin:/Library/Apple/usr/bin:/Users/sthomas/flut
                            ter/bin"
                            builtin-copy -exclude .DS_Store -exclude CVS
-exclude .svn -exclude .git
                            -exclude .hg -resolve-src-symlinks

/Users/sthomas/Desktop/Flutter/nothing_new/ios/Pods/leveldb-library/incl
                            ude/leveldb/cache.h

/Users/sthomas/Desktop/Flutter/nothing_new/build/ios/Release-iphoneos/le
                            veldb-library/leveldb.framework/Headers
                    
                        CpHeader leveldb-library/include/leveldb/comparator.h

/Users/sthomas/Desktop/Flutter/nothing_new/build/ios/Release-iphoneos/leveld
                        b-library/leveldb.framework/Headers/comparator.h
                            cd
/Users/sthomas/Desktop/Flutter/nothing_new/ios/Pods
                            export

PATH="/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.plat

form/Developer/usr/bin:/Applications/Xcode.app/Contents/Developer/usr/bi

n:/Users/sthomas/Desktop/firebase/y/google-cloud-sdk/bin:/usr/local/bin:

/usr/bin:/bin:/usr/sbin:/sbin:/Library/Apple/usr/bin:/Users/sthomas/flut
                            ter/bin"
                            builtin-copy -exclude .DS_Store -exclude CVS
-exclude .svn -exclude .git
                            -exclude .hg -resolve-src-symlinks

/Users/sthomas/Desktop/Flutter/nothing_new/ios/Pods/leveldb-library/incl
                            ude/leveldb/comparator.h

/Users/sthomas/Desktop/Flutter/nothing_new/build/ios/Release-iphoneos/le
                            veldb-library/leveldb.framework/Headers
                    
                        CpHeader leveldb-library/include/leveldb/db.h

/Users/sthomas/Desktop/Flutter/nothing_new/build/ios/Release-iphoneos/leveld
                        b-library/leveldb.framework/Headers/db.h
                            cd
/Users/sthomas/Desktop/Flutter/nothing_new/ios/Pods
                            export

PATH="/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.plat

form/Developer/usr/bin:/Applications/Xcode.app/Contents/Developer/usr/bi

n:/Users/sthomas/Desktop/firebase/y/google-cloud-sdk/bin:/usr/local/bin:

/usr/bin:/bin:/usr/sbin:/sbin:/Library/Apple/usr/bin:/Users/sthomas/flut
                            ter/bin"
                            builtin-copy -exclude .DS_Store -exclude CVS
-exclude .svn -exclude .git
                            -exclude .hg -resolve-src-symlinks

/Users/sthomas/Desktop/Flutter/nothing_new/ios/Pods/leveldb-library/incl
                            ude/leveldb/db.h

/Users/sthomas/Desktop/Flutter/nothing_new/build/ios/Release-iphoneos/le
                            veldb-library/leveldb.framework/Headers
                    
                        CpHeader leveldb-library/include/leveldb/dumpfile.h

/Users/sthomas/Desktop/Flutter/nothing_new/build/ios/Release-iphoneos/leveld
                        b-library/leveldb.framework/Headers/dumpfile.h
                            cd
/Users/sthomas/Desktop/Flutter/nothing_new/ios/Pods
                            export

PATH="/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.plat

form/Developer/usr/bin:/Applications/Xcode.app/Contents/Developer/usr/bi

n:/Users/sthomas/Desktop/firebase/y/google-cloud-sdk/bin:/usr/local/bin:

/usr/bin:/bin:/usr/sbin:/sbin:/Library/Apple/usr/bin:/Users/sthomas/flut
                            ter/bin"
                            builtin-copy -exclude .DS_Store -exclude CVS
-exclude .svn -exclude .git
                            -exclude .hg -resolve-src-symlinks

/Users/sthomas/Desktop/Flutter/nothing_new/ios/Pods/leveldb-library/incl
                            ude/leveldb/dumpfile.h

/Users/sthomas/Desktop/Flutter/nothing_new/build/ios/Release-iphoneos/le
                            veldb-library/leveldb.framework/Headers
                    
                        CpHeader leveldb-library/include/leveldb/env.h

/Users/sthomas/Desktop/Flutter/nothing_new/build/ios/Release-iphoneos/leveld
                        b-library/leveldb.framework/Headers/env.h
                            cd
/Users/sthomas/Desktop/Flutter/nothing_new/ios/Pods
                            export

PATH="/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.plat

form/Developer/usr/bin:/Applications/Xcode.app/Contents/Developer/usr/bi

n:/Users/sthomas/Desktop/firebase/y/google-cloud-sdk/bin:/usr/local/bin:

/usr/bin:/bin:/usr/sbin:/sbin:/Library/Apple/usr/bin:/Users/sthomas/flut
                            ter/bin"
                            builtin-copy -exclude .DS_Store -exclude CVS
-exclude .svn -exclude .git
                            -exclude .hg -resolve-src-symlinks

/Users/sthomas/Desktop/Flutter/nothing_new/ios/Pods/leveldb-library/incl
                            ude/leveldb/env.h

/Users/sthomas/Desktop/Flutter/nothing_new/build/ios/Release-iphoneos/le
                            veldb-library/leveldb.framework/Headers
                    
                        CpHeader leveldb-library/include/leveldb/export.h

/Users/sthomas/Desktop/Flutter/nothing_new/build/ios/Release-iphoneos/leveld
                        b-library/leveldb.framework/Headers/export.h
                            cd
/Users/sthomas/Desktop/Flutter/nothing_new/ios/Pods
                            export

PATH="/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.plat

form/Developer/usr/bin:/Applications/Xcode.app/Contents/Developer/usr/bi

n:/Users/sthomas/Desktop/firebase/y/google-cloud-sdk/bin:/usr/local/bin:

/usr/bin:/bin:/usr/sbin:/sbin:/Library/Apple/usr/bin:/Users/sthomas/flut
                            ter/bin"
                            builtin-copy -exclude .DS_Store -exclude CVS
-exclude .svn -exclude .git
                            -exclude .hg -resolve-src-symlinks

/Users/sthomas/Desktop/Flutter/nothing_new/ios/Pods/leveldb-library/incl
                            ude/leveldb/export.h

/Users/sthomas/Desktop/Flutter/nothing_new/build/ios/Release-iphoneos/le
                            veldb-library/leveldb.framework/Headers
                    
                        CpHeader leveldb-library/include/leveldb/filter_policy.h

/Users/sthomas/Desktop/Flutter/nothing_new/build/ios/Release-iphoneos/leveld
                        b-library/leveldb.framework/Headers/filter_policy.h
                            cd
/Users/sthomas/Desktop/Flutter/nothing_new/ios/Pods
                            export

PATH="/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.plat

form/Developer/usr/bin:/Applications/Xcode.app/Contents/Developer/usr/bi

n:/Users/sthomas/Desktop/firebase/y/google-cloud-sdk/bin:/usr/local/bin:

/usr/bin:/bin:/usr/sbin:/sbin:/Library/Apple/usr/bin:/Users/sthomas/flut
                            ter/bin"
                            builtin-copy -exclude .DS_Store -exclude CVS
-exclude .svn -exclude .git
                            -exclude .hg -resolve-src-symlinks

/Users/sthomas/Desktop/Flutter/nothing_new/ios/Pods/leveldb-library/incl
                            ude/leveldb/filter_policy.h

/Users/sthomas/Desktop/Flutter/nothing_new/build/ios/Release-iphoneos/le
                            veldb-library/leveldb.framework/Headers
                    
                    
[ +869 ms] "flutter ios" took 53,579ms.
Encountered error while building for device.

#0      throwToolExit (package:flutter_tools/src/base/common.dart:14:3)
#1      BuildIOSCommand.runCommand
(package:flutter_tools/src/commands/build_ios.dart:93:7)
<asynchronous suspension>
#2      FlutterCommand.verifyThenRunCommand
(package:flutter_tools/src/runner/flutter_command.dart:722:18)
#3      _rootRunUnary (dart:async/zone.dart:1192:38)
#4      _CustomZone.runUnary (dart:async/zone.dart:1085:19)
#5      _FutureListener.handleValue (dart:async/future_impl.dart:141:18)
#6      Future._propagateToListeners.handleValueCallback
(dart:async/future_impl.dart:682:45)
#7      Future._propagateToListeners (dart:async/future_impl.dart:711:32)
#8      Future._completeWithValue (dart:async/future_impl.dart:526:5)
#9      Future._asyncComplete.<anonymous closure>
(dart:async/future_impl.dart:556:7)
#10     _rootRun (dart:async/zone.dart:1184:13)
#11     _CustomZone.run (dart:async/zone.dart:1077:19)
#12     _CustomZone.runGuarded (dart:async/zone.dart:979:7)
#13     _CustomZone.bindCallbackGuarded.<anonymous closure>
(dart:async/zone.dart:1019:23)
#14     _microtaskLoop (dart:async/schedule_microtask.dart:43:21)
#15     _startMicrotaskLoop (dart:async/schedule_microtask.dart:52:5)
#16     _runPendingImmediateCallback
(dart:isolate-patch/isolate_patch.dart:118:13)
#17     _RawReceivePortImpl._handleMessage
(dart:isolate-patch/isolate_patch.dart:169:5)
```

**Comment by SeanOThomas** [comment]
<img width="270" alt="Screen Shot 2020-04-06 at 11 32 34 PM" src="https://user-images.githubusercontent.com/52874374/78554544-71852980-785f-11ea-896c-b9ab3bcf3a48.png">

**Comment by iapicca** [comment]
Hi again @SeanOThomas 
the logs can be lengthy could you please attach it as txt file?
thank you

**Comment by SeanOThomas** [comment]
@iapicca I've attached the whole txt file for **flutter build ios --verbose**

```
Encountered error while building for device.

#0      throwToolExit (package:flutter_tools/src/base/common.dart:14:3)
#1      BuildIOSCommand.runCommand (package:flutter_tools/src/commands/build_ios.dart:93:7)
<asynchronous suspension>
#2      FlutterCommand.verifyThenRunCommand (package:flutter_tools/src/runner/flutter_command.dart:722:18)
#3      _rootRunUnary (dart:async/zone.dart:1192:38)
#4      _CustomZone.runUnary (dart:async/zone.dart:1085:19)
#5      _FutureListener.handleValue (dart:async/future_impl.dart:141:18)
#6      Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:682:45)
#7      Future._propagateToListeners (dart:async/future_impl.dart:711:32)
#8      Future._completeWithValue (dart:async/future_impl.dart:526:5)
#9      Future._asyncComplete.<anonymous closure> (dart:async/future_impl.dart:556:7)
#10     _rootRun (dart:async/zone.dart:1184:13)
#11     _CustomZone.run (dart:async/zone.dart:1077:19)
#12     _CustomZone.runGuarded (dart:async/zone.dart:979:7)
#13     _CustomZone.bindCallbackGuarded.<anonymous closure> (dart:async/zone.dart:1019:23)
#14     _microtaskLoop (dart:async/schedule_microtask.dart:43:21)
#15     _startMicrotaskLoop (dart:async/schedule_microtask.dart:52:5)
#16     _runPendingImmediateCallback (dart:isolate-patch/isolate_patch.dart:118:13)
#17     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:169:5)
```
[ios_release_logs.txt](https://github.com/flutter/flutter/files/4440989/ios_release_logs.txt)

**Comment by SeanOThomas** [comment]
@iapicca do you have an update? Sorry to be urgent, but I'm blocked by this.

**Comment by iapicca** [comment]
Hi @SeanOThomas 
are you still experiencing the issue with the latest stable or above?
Thank you

**Comment by urvi-aecor** [comment]
I am facing same issue. My configurations are 
Flutter 1.17.5 • channel stable • https://github.com/flutter/flutter.git
Framework • revision 8af6b2f038 (2 days ago) • 2020-06-30 12:53:55 -0700
Engine • revision ee76268252
Tools • Dart 2.8.4

**Comment by TahaTesser** [comment]
Without additional information we are unfortunately not sure how to resolve this issue. 
We are therefore reluctantly going to close this bug for now. 
Please don't hesitate to comment on the bug if you have any more information for us; we will reopen it right away!  
Thanks for your contribution.

Could everyone who still has this problem please file a new issue with the exact description of what happens, logs and the output of `flutter doctor -v`.
All system setups can be slightly different, so it's always better to open new issues and reference related issues.
